Invite.cpp

```diff
@@ -0,0 +1,129 @@
+#include "Invite.hpp"
+#include <vector>
+
+Invite::Invite() : ACommand()
+{
+	/*this is forbidden */
+}
+
+Invite::~Invite()
+{
+#if SHOW_CONSTRUCTOR
+	std::cout << "Invite destructor" << std::endl;
+#endif
+}
+
+Invite::Invite(Server &server, Client &user, std::string rawInput) : ACommand(server, user, rawInput)
```

Comment:
Arg... could you please change user to author please ? xD Sorryyyyyyy

Reply:
fixed!